internal/server/adapters/encrypter/init.go

```diff
@@ -0,0 +1,61 @@
+package encrypter
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/maybecoding/keep-it-safe/internal/server/adapters/encrypter/rotator"
+)
+
+// Init - initialize Encrypter.
+func (e *Encrypter) Init() error {
+	// Check encryption key size
+	switch e.cfg.KeySize {
+	default:
+		return fmt.Errorf("encrypter - Init - incorrect Key Size %d. Must be 16, 24 or 32", e.cfg.KeySize)
+	case 16, 24, 32:
+	}
+
+	// Convert hex msk to bytes
+	msk, err := hex.DecodeString(e.cfg.MasterKeyHex)
+	if err != nil {
+		return fmt.Errorf("encrypter - Init - hex.DecodeString: %w", err)
+	}
+
+	// if master key more then 32 bytes - error
+	if len(msk) > 32 {
+		return fmt.Errorf("encrypter - Init - master key size %d: must be max 32 byte", len(msk))
+	}
+
+	// if key len not in 16, 24 or 32 bytes adds padding
+	switch len(msk) {
+	default:
+		for _, size := range []int{16, 24, 32} {
+			if len(msk) < size {
+				msk = append(msk, make([]byte, size-len(msk))...)
+				fmt.Printf("master key size %d was padded to %d", len(msk), size)
+				break
+			}
+		}
+	case 16, 24, 32:
+	}
```

Comment:
сложно читается и свитч тут явно не к месту, для свитча выше тоже актуально. Кроме того можно вынести в отдельную функцию
```suggestion
func padMasterKey(msk []byte) []byte {
    desiredSizes := []int{16, 24, 32}
    currentLength := len(msk)
    
    for _, size := range desiredSizes {
        if currentLength == size {
            return msk
        } else if currentLength < size {
            padding := make([]byte, size-currentLength)
            msk = append(msk, padding...)
            fmt.Printf("master key size %d was padded to %d\n", currentLength, size)
            return msk
        }
    }
    
    return msk
}
```